build_projects/shared-build-targets-utils/Utils/Crossgen.cs

```diff
@@ -153,10 +153,15 @@ public void CrossgenDirectory(BuildTargetContext c, string pathToAssemblies)
 
                 IList<string> crossgenArgs = new List<string> {
                     "-readytorun", "-in", file, "-out", tempPathName,
-                    "-JITPath", GetLibCLRJitPathForVersion(),
                     "-platform_assemblies_paths", platformAssembliesPaths
                 };
 
+                if (CurrentPlatform.IsUnix)
+                {
+                    crossgenArgs.Add("-JITPath");
```

Comment:
@eerhardt @ellismg @brthor We also need to ensure that the respective platform installers (currently in CLI repo and soon in setup repo) also package the JIT binary.